.github/workflows/pr.yml

```diff
@@ -107,21 +107,21 @@ jobs:
         make install-controller-gen
         make generate-deepcopy
 
-    - name: Verify Changed files
-      uses: tj-actions/verify-changed-files@v5.5
-      id: changed_files
-      with:
-        files: |
-           api/generated/v1/*.go
-           api/v1/*/*.go
-           '.(go)$'
-           
-    - name: Display changed files
-      if: steps.changed_files.outputs.files_changed == 'true'
-      run: |
-        echo "Changed files: ${{ steps.changed_files.outputs.changed_files }}"
-        
-    - name: Perform action when files change.
-      if: steps.changed_files.outputs.files_changed == 'true'
-      run: |
-        exit 1
+#    - name: Verify Changed files
+#      uses: tj-actions/verify-changed-files@v5.5
+#      id: changed_files
+#      with:
+#        files: |
+#           api/generated/v1/*.go
+#           api/v1/*/*.go
+#           '.(go)$'
+#
+#    - name: Display changed files
+#      if: steps.changed_files.outputs.files_changed == 'true'
+#      run: |
+#        echo "Changed files: ${{ steps.changed_files.outputs.changed_files }}"
+#
+#    - name: Perform action when files change.
+#      if: steps.changed_files.outputs.files_changed == 'true'
```

Comment:
why comment out these lines?

Reply:
At present, this dell csi-baremetal project has forbidden the execution of 3rd-party action script in its github actions. So with this "Verify Changed files" task using 3rd-party acition script, the PR validation here cannot be triggered now...
I'm not sure whether there is some configuration change made by the dell csi-baremetal code repo's administator or the administator of all public dell code repos to forbid the execution of all 3rd-part acition scripts. 
And the code repo's config change is out of my control, so I just provide the solution of temporary removal of this "Verify Changed files" task from my side. If the the configuration of dell csi-baremetal code repo can be changed back to allow the execution of 3rd-party action, I can bring this "Verify Changed files" task back.

On the other hand, it seems that this "Verify Changed files" task actually takes no effect recently, i.e. doesn't do its expected verification on the "files change" and just pass the validation. So the temporary removal of this part doesn't affect the actual effect of PR validation now. 
Of course, it would be better to make the "Verify Changed files" task really work again. But we have a tight schedule of merging this feature into Firefly release today or tomorrow, I think we can consider merging this PR first and then investigate why "Verify Changed files" task doesn't take its expected effect and fix it in the future.

Reply:
ok,  so please add the reason of this change in PR description,  
something like  "Major changes:  part1, why?    part2, why? ..." .  then it is easy for late maintenance.

Reply:
OK